bin/p-s.js

```diff
@@ -55,7 +56,23 @@ runPackageScript({
 function getPSConfig() {
   var psConfigFilename = program.config ? resolve(process.cwd(), program.config) : findUp.sync('package-scripts.js')
   try {
-    return require(psConfigFilename)
+    // Require a module like babel-register
+    if (program.require) {
+      try {
+        require(program.require)
+      } catch (e) {
+        log.warn({
+          message: colors.yellow('Unable to preload ' + program.require),
+          ref: 'preload-module'
+        })
+      }
+    }
+
+    var config = require(psConfigFilename)
+    if (program.require === 'babel-register') {
```

Comment:
Could we actually do:

```javascript
if (config.default) {
  config = config.default
}
```

That should be safer and allow people to use ES6 modules or CommonJS even if they do transpile with babel or anything else.